e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -329,6 +357,26 @@ describe("scenarios > embedding > dashboard parameters", () => {
     getDashboardCard()
       .findByText("Practical Bronze Computer")
       .should("be.visible");
+
+    cy.log("test downloading result (metabase#36721)");
+    getDashboardCard().realHover();
+    downloadAndAssert(
+      {
+        fileType: "csv",
+        isDashboard: true,
+        isEmbed: true,
+        logResults: true,
+        downloadUrl: "/api/embed/dashboard/*/dashcard/*/card/*/csv*",
+      },
+      sheet => {
```

Comment:
This is what the result looks like
![image](https://github.com/metabase/metabase/assets/1937582/1054a570-d72c-4dfb-9ea1-627d8fb3d869)